docs/en/stack/ml/df-analytics/ml-lang-ident.asciidoc

```diff
@@ -0,0 +1,191 @@
+[role="xpack"]
+[[ml-lang-ident]]
+=== {lang-ident-cap}
```

Comment:
IMO this doesn't really feel like a description of a concept, since it gets into implementation details and examples quite quickly.  My inclination would be to have maybe a paragraph of introduction to this concept of language identification at the end of the inference page (http://stack-docs_816.docs-preview.app.elstc.co/guide/en/machine-learning/master/ml-inference.html) and then have all the details (ideally covering the whole pipeline setup, for example) in this section: http://stack-docs_816.docs-preview.app.elstc.co/guide/en/machine-learning/master/dfanalytics-examples.html

Reply:
@lcawl 
The aim is to open a "Trained models" section when we'll have more than one trained models. For this reason, I wouldn't split the content between the Inference conceptual docs and the Examples because it would make harder to merge them together when we'll have the next trained model.
I'll add a short paragraph to the inference conceptual docs about lang identification with a link that points to this piece, so there will be continuity between the two texts.